core/config_utils.go

```diff
@@ -67,20 +67,20 @@ func getServicePropertiesFromCredentialFile(credentialKey string) map[string]str
 	// Check the search order for the credential file that we'll attempt to load:
 	var credentialFilePath string
 
-	// 1) ${IBM_CREDENTIALS_FILE}
-	envPath := os.Getenv(IBM_CREDENTIAL_FILE_ENVVAR)
-	if _, err := os.Stat(envPath); err == nil {
-		credentialFilePath = envPath
-	}
-
-	// 2) <user-home-dir>/ibm-credentials.env
+	// 1) <user-home-dir>/ibm-credentials.env
```

Comment:
The order should be:
1. ${IBM_CREDENIALS_FILE}
2. <current-working-directory>/ibm-credentials.env
3. <home-dir>/ibm-credentials.env
You moved <home-dir> to the top.

Reply:
Oh I read the description wrong, correcting it